overview/saml.mdx

```diff
@@ -15,6 +15,45 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+
+
+1. In the Organization Settings for your project, make sure that SAML is enabled and that all orgs can setup SAML.
+
+![organization settings](/images/saml-setup-1.png)
+
+2. Under Users, make sure that there is at least one test user with Admin or Owner priviledges.
+
+![test user](/images/saml-setup-2.png)
+
+3. On the dashboard for your project, click the "View Org Page" button under the Test Environment.
+
+![dashboard](/images/saml-setup-3.png)
+
+4. Click the Create Organization button.
+
+![create organization](/images/saml-setup-4.png)
+
+5. Write a name for the organization, and click Create Organization.
```

Comment:
(if it's annoying to change the picture, feel free to change Acme above to Test Org)